buildspecs/linux_390-64.spec

```diff
@@ -134,6 +134,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="gc_debugAsserts" value="true"/>
 		<flag id="gc_idleHeapManager" value="true"/>
 		<flag id="gc_inlinedAllocFields" value="true"/>
+		<flag id="gc_enableDoubleMap" value="true"/>
```

Comment:
Has this been tested on all the platforms it's being enabled on?

Reply:
Yes, all of them have been tested with and without double map